free-1.md

```diff
@@ -1,68 +1,198 @@
+reported right away, and the current drawing would remain unchanged.
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
 
+It should be easy (and should feel natural) to make simple geometric drawings
+and format them in simple ways (rotation, size, cropping, labelling, etc).
+
+It should be hard to make really complex drawings with lots of parts. Visual
+clutter is not something that is desirable in geometry.
+
+It should be impossible/extremely difficult to simulate a Turing machine. This
+language is not for computation. In the end it is an assistive formatting
+language.
+
+It would also be cool to allow for sub-routines. People should be able to
+define, "Find the centroid of this triangle" and re-use that idea without
+having to redo all the work.
 
 ### Related work
-_Are there any other DSLs in this domain? If not, describe how you know there
-aren't and conjecture why not. If so, describe them and provide links. How well 
-do they address the need? Are there any particularly admirable qualities of the
-language? Are there parts of the language you think could be improved?_
 
+There are a lot of interesting graphical drawing aides.
+
+For example [Sketchometry](http://start.sketchometry.org/). These all do well
+in that they are graphical and are thus easy to start using, but they run into
+problems with when geometrically distinct ideas are spatially close (as
+discussed above).
+
+There is also the `tkx-euclid` LaTeX package. It seems to be very complete, and
+has seems to have a good underlying geometric model, but it is somewhat
+cryptic, doesn't look _that much_ like geometric writing, and it is sometimes
+unclear what options the user has (Did the thing I drew intersect with
+something else?).
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
 
 ### Suitability
-_If someone were to work on this project, what percentage of their time would be
-spent directly engaging in the **language** aspects of this project (e.g.,
-making language design decisions), as opposed to "systems" aspects of the
-project (e.g., implementing a complicated semantics that doesn't require a lot
-of language design)?_
 
+There is some question here. I think that the time would probably end up being
+split 50/50, although that would change depending on whether I could find a
+good geometry library that would meet most of my needs (shape drawing and
+intersection detection come to mind right now, but I imagine the list will
+grow). The language design time would be spent thinking about syntax for use
+input, thinking about how the computer will provide feedback to the users, and
+thinking about the users should be able to incorporate that feedback into
+further instructions.
+
+If I were building this from the ground up this would probably be way to
+**systems** heavy.
```

Comment:
It does seem like it would take a lot of work to build the system; however I think in order to build a powerful yet intuitive language it would take a lot of work as well.